mbtb_app/web_portal/mbtb_portal/assets/js/suspend_user_with_reason.js

```diff
@@ -0,0 +1,28 @@
+var suspend_user_with_reason = angular.module("suspend_user_with_reason_app", []);
+
+suspend_user_with_reason.controller("suspend_user_with_reason_controller",
+  ['$scope', '$window', '$http', function ($scope, $window, $http) {
+
```

Comment:
8 -> Here, you need to bind `request_ids_` from ejs view to angular app, so that we can send it with post request.

```
// binding data to angular variable from ejs view
$scope.request_ids_ = $window.request_ids_;
```